mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -248,8 +248,8 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 		ARG 1 source
 	METHOD method_6062 isImmobile ()Z
 	METHOD method_6063 getMaximumHealth ()F
-	METHOD method_6064 getNextBreathInAir (I)I
-		ARG 1 breath
+	METHOD method_6064 updateAirOnLand (I)I
```

Comment:
This method has no side effect, I do not think `update` is a good fit. It simply computes the amount of air using the given current value.